concordium-sdk/src/main/java/com/concordium/sdk/responses/accountinfo/credential/Key.java

```diff
@@ -1,20 +1,55 @@
 package com.concordium.sdk.responses.accountinfo.credential;
 
+import com.concordium.sdk.transactions.PublicKey;
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Getter;
 import lombok.ToString;
+import lombok.val;
+import org.apache.commons.codec.DecoderException;
+import org.apache.commons.codec.binary.Hex;
+
+import java.nio.ByteBuffer;
 
 @Getter
 @ToString
 public final class Key {
     private final String verifyKey;
     private final String schemeId;
 
+    private final String ED25519 = "ed25519";
```

Comment:
Maybe `ED25519PublicKey` could be used instead?

I don't see any benefits from having `this`, `PublicKey` below and `ED25519PublicKey` representing the same type.

Reply:
Changed, the flow to use ED25519PublicKey